### DISCUSSION TITLE
Div. Listen fehlerhaft?

### DISCUSSION THREAD

**Discussion by usery787** [RPiList/specials#1300] [General]
Moin,

mir ist gerade aufgefallen das bei einigen Listen der Inhalt nicht erkannt wird.
z.B.
[notserious](https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/notserious)
[streaming](https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/Streaming)
[gambling](https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/gambling)
[fake-science](https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/Fake-Science)
[corona-blocklist](https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/Corona-Blocklist)
[domainsquatting1](https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/DomainSquatting1)
[domainsquatting2](https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/DomainSquatting2)
[domainsquatting3](https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/DomainSquatting3)
[domainsquatting4](https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/DomainSquatting4)
[Sparkasse/Niedersachsen1](https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/DomainSquatting/DE/Sparkasse/Niedersachsen1)
[Sparkasse/Niedersachsen2](https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/DomainSquatting/DE/Sparkasse/Niedersachsen2)

Bei den genannten Listen ist unter Adlists im Pihole und dann unter details, "Number of domains on this list:" werden keine Domains erkannt. (s. Screenshot) Scheinbar liegt es an den Zeichen "||" die jeweils vor jeder Domain sind. Andere Listen wo die Zeichen nicht sind, werden erkannt.

Ist das Problem bekannt, oder aus irgendwelchen Gründen so gewollt?

EDIT:
Wer lesen kann ist natürlich klar im Vorteil...
[HIER](https://github.com/RPiList/specials/discussions/1090) wurde das Thema ja besprochen, erklärt aber nicht warum unter details "Number of domains on this list:" keine erkannt werden.

![Screenshot 2023-10-08 095159](https://github.com/RPiList/specials/assets/120797219/04c04ac3-c834-4c9c-a3c7-05aa387bf390)

**Comment by RPiList** [comment]
Hi,
die Lösung ist, dass Ihr Pihole veraltet ist. Ein Pihole von vor März 2023 erkennt die "neuen" Einträge nicht. Einfach updaten... dann werden die "neuen" Domains als ABP Domains erkannt und auch in der Übersicht richtig gezeigt. Viele Grüße

**Comment by usery787** [comment]
Moin,
das Pihole habe ich vor 2 Wochen frisch installiert, allerdings habe ich die Adlists als Backup hinzugefügt. Vllt ist dass das Problem....

**Comment by RPiList** [comment]
Es gibt in der Tat einige Listen die inzw. "leer" sind, weil Sie nach dem ABP-Update nicht mehr benötigt wurden. Wir behalten sie aber, weil die früher oder später wieder gebraucht werden.

Aber die notserious Liste in Ihrem Beispiel ist keine davon. 

![image](https://github.com/RPiList/specials/assets/56664851/6f4d954f-2d9c-43a5-9547-f1a70c0c2078)

So sollte es aussehen, wenn ein Update für die notserious Liste durchgeführt wird. Da ist dann der Hinweis auf die ABP Domains zu finden.
